furnace/related_worker.py

```
@@ -0,0 +1,151 @@
+    for i,result in enumerate(search.results()):
+        # print(result._get_default_filename)
+        # if int(result.published.year)>=2022:
+        # if check_relevant(result.title,result.summary,['CLIP','knowledge distillation']) == 'Y':
+        print(i, result.title, result.entry_id, result.published, )
+        print(result.summary)
+        print('\n\n')
+        rst.append(result)
+    return
+
+
+rst = simple_query()
```

Ruff (0.8.2)

Undefined name `simple_query`

(F821)